bims/api_views/location_site.py

```diff
@@ -289,6 +294,138 @@ def get(self, request):
             return Response(response_data)
 
 
+    def get_biodiversity_data(self, collections):
+        biodiversity_data = {}
+        taxa = self.get_origin_cons_endemsim_data(collections)
+        # If I found more than one class of animal
+        if 'Actinopterygii' not in taxa:
```

Comment:
This has a bad smell...

Reply:
Agree we shouldn’t hardcoded the species name.

Reply:
Thanks, saw below how to reference this. Will fix.